reference/api/search.md

```diff
@@ -460,7 +460,17 @@ Optionally, you can indicate a custom crop length for any attributes given to `a
 
 Instead of supplying individual attributes, you can provide `["*"]` as a wildcard: `attributesToCrop=["*"]`. This causes `_formatted` to include the cropped values of all attributes present in [`attributesToRetrieve`](#attributes-to-retrieve).
 
-**Meilisearch crops around the first occurrence of any one of the terms present in the search query.** If Meilisearch does not find any query terms in a field, cropping begins at the first word in that field.
+#### Cropping rules
+
+Suppose you have a field containing the following string: `Donatello is a skilled and smart turtle. Leonardo is the most skilled turtle. Raphael is the strongest turtle.`
+
+Meilisearch respects sentence boundaries when cropping. For example, if your search term is `Leonardo` and your `cropLength` is 6, Meilisearch will prioritize keeping the sentence together and return: `Leonardo is the most skilled turtle.`
+
+If a query contains only a single search term, Meilisearch crops around the first occurrence of that term. If you search for `turtle` and your `cropLength` is 7, Meilisearch will return the first instance of that word: `Donatello is a skilled and smart turtle.`
```

Comment:
Is this right?

I expected it to return `... skilled turtle. Raphael is the strongest turtle." since that has two instances of the query term, and they're closer together than the previous two "turtle"s.

@gmourier @ManyTheFish 👀

Reply:
I _think_ sentence context wins in this case, but that's a very good point. I'll get in touch with @ManyTheFish once he's back.

Reply:
Hello @dichotommy @guimachiavelli.

@dichotommy should be right because `... skilled turtle. Raphael is the strongest turtle.` contains 2 matches, where `Donatello is a skilled and smart turtle.` contains only one 😕

Reply:
Thanks for clearing this up, @ManyTheFish.

Perhaps the easiest way to make this accurate and illustrative is to reduce `cropLength` to 5. That should return `… a skilled and smart turtle`, correct? (There are no instances of `turtle` within 5 words of each other, so I _think_ all instances of turtle have the same weight.)

Reply:
After testing this in the latest RC (v0.28.0rc2), it seems the original example was correct. 

Even though `turtle` does appear two times in `… skilled turtle. Raphael is the strongest turtle.`, the algorithm looks for _unique_ matches in the cropped segment. In this situation, Meilisearch keeps the first match and ignores the other instances of `turtle`.

I have confirmed this is the expected behaviour with core and am marking the suggestion as resolved.